src/classes/SObjectRepositoryMocks.cls

```diff
@@ -0,0 +1,122 @@
+/*************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.             *
+* See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
+*************************************************************************************************/
+@isTest
+public class SObjectRepositoryMocks {
+
+    public virtual class Base extends DMLMock.Base implements ISObjectRepository {
```

Comment:
This class seems to have grown very large and is becoming difficult to maintain. Please consider refactoring to reduce the number of methods.